src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const App = () => {
```

Comment:
The Google styleguide suggests to comment the functions, but this type of functions will be recurrent, and they behave in a very special way since they're intended to be React Components.

I don't know if it's okay to document them as normal functions